quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java

```diff
@@ -40,6 +43,11 @@ public class QuarkusCliExtensionsIT {
     @Inject
     static QuarkusCliClient cliClient;
 
+    @BeforeAll
+    static void useRHBQRegistry() {
+        cliClient.run("registry", "add", "registry.quarkus.redhat.com");
```

Comment:
Quarkus product coverage is done by Jenkins, and the environment is already setup by our jobs, so this `BeforeAll` should be removed

Reply:
with `BeforeAll`, you are able to run the tests f.e. with `mvn clean verify -pl quarkus-cli/ -Dit.test=QuarkusCliExtensionsIT -Dquarkus.platform.version=2.7.5.Final-redhat-00011 -Dquarkus-plugin.version=2.7.5.Final-redhat-00011 -Dquarkus.platform.group-id=com.redhat.quarkus.platform -Dquarkus.platform.artifct-id=quarkus-bom -Dmaven.repo.local=/home/mvavrik/rhbq/275CR1/rh-quarkus-2.7.5.GA-maven-repository/maven-repository`.

In my understanding, running Test Suite branch 2.7 with RHBQ is more than expected, so it's prerequisite to set registry and should be documented in a same fashion as other in README.